6.1/Dockerfile

```diff
@@ -51,4 +51,5 @@ RUN echo '/usr/local/lib64' > /etc/ld.so.conf.d/local-lib64.conf \
 RUN set -x \
 	&& dpkg-divert --divert /usr/bin/gcc.orig --rename /usr/bin/gcc \
 	&& dpkg-divert --divert /usr/bin/g++.orig --rename /usr/bin/g++ \
+	&& dpkg-divert --divert /usr/bin/gfortran.orig --rename /usr/bin/gfortran \
```

Comment:
Is `/usr/bin/gfortran` installed in the image already at this point?  (it's not part of `buildpack-deps:jessie` like `gcc` and `g++` are)